src/ensembl/src/content/app/browser/track-panel/TrackPanel.tsx

```diff
@@ -83,10 +78,33 @@ const TrackPanel: FunctionComponent<TrackPanelProps> = (
     }
   }, [props.breakpointWidth, props.toggleTrackPanel]);
 
+  const [trackAnimation, setTrackAnimation] = useSpring(() => ({
+    config: { tension: 280, friction: 45 },
+    height: '100%',
+    position: 'absolute' as 'absolute',
+    display: 'block',
+    left: 'calc(-356px + 100vw)'
+  }));
+
+  const getbrowserWidth = (): string => {
+    if (props.drawerOpened) {
+      return 'calc(41px + 0vw)';
+    }
+    return props.trackPanelOpened
+      ? 'calc(-356px + 100vw)'
+      : 'calc(-36px + 100vw)';
+  };
+
+  useEffect(() => {
+    setTrackAnimation({
+      left: getbrowserWidth()
```

Comment:
Can you change `getbrowserWidth` to `getBrowserWidth`?

Reply:
That's done